rust/main/agents/validator/src/settings.rs

```diff
@@ -274,7 +276,7 @@ fn parse_checkpoint_syncer(syncer: ValueParser) -> ConfigResult<CheckpointSyncer
             cfg_unwrap_all!(&syncer.cwp, err: [bucket, region]);
             err.into_result(CheckpointSyncerConf::S3 {
                 bucket,
-                region,
+                region: Region::new(region),
```

Comment:
any reason not to parse `Region` with `rusoto` in here as well?

Reply:
I didn't want to add it as a dep to the validator but might as well, I'll add